### ISSUE TITLE
PR#3 updated region retrieval rates to the wrong values

### ISSUE THREAD

**Issue by jwm** [liangzan/aws-glacier-calculator#4]
Hi, thanks for maintaining this calculator!

I noticed that the results I was getting didn't match the official AWS calculator (http://calculator.s3.amazonaws.com/), and it seems to come down to the retrieval rates.

It seems like https://github.com/liangzan/aws-glacier-calculator/commit/e7547411dea1c2099d3dcff4f1fdd93a8a8c0632 updated them to the wrong values. The examples I see in the Amazon Glacier FAQ all use $0.01/GB. Try as I might, I couldn't seem to find the per-region prices for Glacier retrieval.

If I take this calculator's results and adjust them for the $0.01/GB retrieval rate (in `us-east-1`), my results line up with the official AWS calculator _and_ with the rudimentary calculation shown by the Glacier "Data Retrieval Policy" screen in the AWS Management Console. 

@karpach Are you sure that the retrieval rates in your PR are current?

**Comment by karpach** [comment]
I got my prices from the [official amazon glacier page](https://aws.amazon.com/glacier/pricing/):

![Prices per region screenshot](https://i.imgur.com/8Hqs05g.png)

**Comment by jwm** [comment]
@karpach As an example, in your commit (https://github.com/liangzan/aws-glacier-calculator/commit/e7547411dea1c2099d3dcff4f1fdd93a8a8c0632), you changed the retrieval pricing in `regionRetrievalRate()` for `us-west-2` from $0.011 to 0.055. Based on your screenshot, that value should have remained unchanged.

**Comment by karpach** [comment]
You are right. Looks like number of the requests is not taken into account by this calculator. I'll submit a fix.

**Comment by jwm** [comment]
Thanks @karpach!
